src/components/StartFooter/StartFooter.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Container,
+  Spacer,
+  useTheme,
+} from 'react-neu'
+
+import Split from 'components/Split'
+
+const StartFooter: React.FC = () => {
+  const { darkMode } = useTheme()
+  return (
+    <StyledFooter darkMode={darkMode}>
+      <Spacer size="lg" />
+      <Container>
+        <Split>
+          <StyledLink href="https://discord.gg/nKKhBbk">Discord</StyledLink>
+          <StyledLink href="https://github.com/yam-finance/yam-www">Github</StyledLink>
+          <StyledLink href="https://twitter.com/YamFinance">Twitter</StyledLink>
+          <StyledLink href="https://snapshot.page/#/yam">Proposals</StyledLink>
+          <StyledLink href="https://forum.yam.finance">Forum</StyledLink>
```

Comment:
can we have all these in a json file with the following form, to be able to use it on both footers:
```json
[
    {
        "name": "Proposals",
        "url": "http....",
        "router": false,
        "homeDisplay": true,
    },
    ...
]
```